### ISSUE TITLE
Can't build with old Rust

### ISSUE THREAD

**Issue by danbst** [katharostech/cast2gif#4]
Rustc 1.39.0

```
   Compiling gif-dispose v2.3.0
error[E0658]: use of unstable library feature 'inner_deref': newly added
  --> /home/danbst/.cargo/registry/src/github.com-1ecc6299db9ec823/gif-dispose-2.3.0/src/screen.rs:74:29
   |
74 |         self.blit(local_pal.as_deref(), frame.dispose,
   |                             ^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/50264
```
Please add also Rust version which it should be built with (yeah, I know I should use nightly, but still would be great for r13y(reproducibility)).

**Comment by zicklag** [comment]
cast2gif itself doesn't use any relatively new features of Rust that I know of, but `gifski` which we use for GIF rendering looks like it needs at least  Rust 1.42+. I'll put that in the README.

**Comment by zicklag** [comment]
Fixed with 8d572ac.
